File: erisp-keyvehicle/src/main/webapp/app/assets/js/controllers/warningDisposal/warningSetService.js
app.factory("warningSetService",["$http",function($http) {
	
		//添加预警
		function addWarning(param,success,error){
			var dataUrl = "/erisp-keyvehicle/service/warningSet/addWarning";
			$http({
				url:dataUrl,
				method:'GET',
				params:param
			}).success(function(data){
				success(data);
			}).error(function(data){
				error(data)
			});
		}
		
		// GET查询预警信息
    	function queryWarningInfoList(params,success,error) {
            var dataUrl = "/erisp-keyvehicle/service/warningSet/queryWarningInfoList";
            $http({
                url : dataUrl,
                method : "GET",
                params : params
            }).success(function(data) {
                success(data);
            }).error(function(data) {
                error(data)
            });
    	}
    	
    	// 删除预警信息
    	function deleteWarningInfo(params,success,error) {
    		var dataUrl = "/erisp-keyvehicle/service/warningSet/deleteWarningInfo";
    		$http({
    			url : dataUrl,
    			method : "POST",
    			params : params 
    		}).success(function(data) {
    			success(data);
    		}).error(function(data) {
    			error(data)
    		});
    	}
    	//修改预警
    	function updateWarning(params,success,error) {
    		var dataUrl = "/erisp-keyvehicle/service/warningSet/updateWarning";
    		$http({
    			url : dataUrl,
    			method : "GET",
    			params : params 
    		}).success(function(data) {
    			success(data);
    		}).error(function(data) {
    			error(data)
    		});
    	}
    	
	    return {
	    	addWarning:addWarning,
	    	updateWarning:updateWarning,
	    	queryWarningInfoList:queryWarningInfoList,
	    	deleteWarningInfo:deleteWarningInfo
	    }
    }
])

File: erisp-keyvehicle/src/main/java/com/gkhb/keyvehicle/service/WarningSetService.java
package com.gkhb.keyvehicle.service;

import java.util.List;

import com.gkhb.keyvehicle.common.model.Page;
import com.gkhb.keyvehicle.model.param.QueryConditionData;
import com.gkhb.keyvehicle.model.view.WarningSetView;

/**
 * 预警设置业务接口
 * @author WeiGuo
 * @data 2017年9月23日下午5:12:45
 */
public interface WarningSetService {

	/**
	 * 添加预警
	 */
	public int addWarning(WarningSetView warningData);
	
	List<WarningSetView> queryWarningInfoList(QueryConditionData queryConditionData,Page page);
	
	int deleteWarningInfo(String id,String warningType);
	
	/**
	 * 修改预警信息
	 * @param warningData
	 * @return
	 */
	public int updateWarning(WarningSetView warningData);
}
